task_manager/app/controllers/tasks_controller.rb

```diff
@@ -2,8 +2,9 @@
 
 class TasksController < ApplicationController
   before_action :set_task_by_id, only: %i[show edit update destroy]
-  def index
-    @tasks = Task.sort_column_direction(params[:sort], params[:direction]).search(params[:keyword_name], Task.progresses[params[:keyword_progress]])
+  def index # rubocop:disable Metrics/AbcSize
+    @tasks = Task.sort_column_direction(params[:sort], params[:direction]).search(params[:keyword_name],
+                                                                                  Task.progresses[params[:keyword_progress]]).page(params[:page]).per(10)
```

Comment:
メソッドチェーン（メソッドの実行結果に対してさらにメソッドを呼び出す）が長くなる場合の改行は以下のようにするのが一般的かなぁと思います。
改行時のドットの位置は行末派と行頭派の派閥があるようですが、ラクマではだいたい行頭になっているので行頭でお願いしたいです。

```suggestion
    @tasks = Task.sort_column_direction(params[:sort], params[:direction])
      .search(params[:keyword_name], Task.progresses[params[:keyword_progress]])
      .page(params[:page]).per(10)
```

Reply:
コメントありがとうございます！
conflictの該当箇所と同様だったのでそちらと同時に修正しました！
https://github.com/Fablic/training/pull/1090/commits/f52611f3426d2d2c95625d31901f0fc9edf2e473